H/highs/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "highs"
```

Comment:
Minor nitpicking: we prefer to use the same name as the upstream project (as long as it's a valid Julia identifier), with the same capitalisation, so this would be
```suggestion
name = "HiGHS"
```
If you change this name here, remember to rename also the directory accordingly, they must match.

Reply:
> If you change this name here, remember to rename also the directory accordingly, they must match.

:wink: